chart/env/staging.yaml

```diff
@@ -211,6 +216,20 @@ api:
       cpu: 1
       memory: "4Gi"
 
+rows:
+  uvicornNumWorkers: "1"
+
+  replicas: 1
+  service:
+    type: NodePort
+  resources:
+    requests:
+      cpu: 100m
+      memory: "512Mi"
```

Comment:
we'll need to increase that number a bit - see (internal) https://huggingface.slack.com/archives/C04L6P8KNQ5/p1688484608969739

Reply:
oh yes. Let's do it in another PR?